modules/ROOT/pages/ts-cannot-process-B2B.adoc

```diff
@@ -0,0 +1,84 @@
+. Once you locate the message flow, click on the message flow card to open it up.
+. From the Message flow configurator, verify the following information per your agreement with your trading partner:
+
+** The correct identifiers are associated with the trading partner.
+
+** The appropriate receive endpoint is configured in the flow.
+
+** The appropriate document type (such as X12, version - 4010, message type - 850) is configured.
+
+** Verify that you have the appropriate acknowledgment configuration.
+
+** Verify that you have an appropriate DataWeave map in the transformation section.
+
+** Verify that you have configured an appropriate target such as both the message type of the target and the associated endpoint configuration.
+
++
+
+. If all of these look good, then ensure that the last deployment for this flow was successful by viewing the latest deployment history on the right side of the flow.
+
+. If you observe a Failed status, try to deploy the message flow to ensure all your configurations have persisted on the Mule runtime apps.
+
+NOTE: It can take a maximum interval of 5 minutes for your most recent changes to be effective on the runtime B2B Mule apps.
+
+=== Verify B2B integration apps
+
+Verify that your B2B integration apps are active.
+
+. From the Anypoint platform, select Anypoint Runtime Manager and *Applications*.
+
+. Verify that the B2B apps on the Mule Runtime are active by looking for a status of `started`. The list of apps includes:
+
+** APIReplicationService
+
+** DocumentProcessingService
+
+** One or more inbound apps. The names of these apps are prefaced with inbound-
+
+** One or more outbound apps. The names of these apps are prefaced with outbound-
+
+** There may be one or more acknowledgment apps. The names of these apps are prefaced with ack-
+
++
+
+. If any of the B2B integration apps do not have a started status, from the Anypoint platform, select Anypoint Runtime Manager and *Applications*.
+
+. For each app that does not have started status, select its row in the table and select *Restart* from the drop-down menu in the upper right of the status area in the information panel.
+
+. If restarting the app does not help you recover the application, select its row in the table and select *Delete* from the drop-down menu in the upper right of the status area in the information panel and confirm the deletion.
+
+. Navigate back to the message flow related to the app you just deleted and deploy the message flow.
+
+Consider the following when performing a delete:
+
+** If you delete the `DocumentProcessingService` or the `APIReplicationService`, deploying any flow results in the redeployment of those apps.
+
+** If you delete any of the inbound and/or outbound and/or ack apps, you have to redeploy the specific flow corresponding to the deleted apps.
+
+If all these scenarios have been verified successfully, and you still cannot deploy message flows, please raise a case on MuleSoft’s customer portal.
```

Comment:
```suggestion
If you still cannot deploy message flows, raise a case on the MuleSoft customer portal.
```
Are we really supposed to send readers to support, and what is the precise name of the portal?